sphinx-doc/module-hpmc-tune.rst

```diff
@@ -16,7 +16,7 @@ hpmc.tune
     :synopsis: Tuners for HPMC.
     :members:
 
-    .. autoclass:: MoveSize(trigger, moves, target, solver, types=None, max_move_size=None)
+    .. autoclass:: MoveSize
```

Comment:
Revert this change please. Due to the indirection used to present `MoveSize` as a _normal_ HOOMD operation rather than a custom action one, sphinx does not know the signature for the class otherwise.

There is a way to tell Python to pretend a class of method has a different signature (yay runtime patching), but we don't currently use that.